protocol/communities/community.go

```diff
@@ -898,6 +901,27 @@ func (o *Community) IsMemberInChat(pk *ecdsa.PublicKey, chatID string) bool {
 	return o.getChatMember(pk, chatID) != nil
 }
 
+// Uses bloom filter members list to estimate presence in the channel.
+// False positive rate is 0.1%.
+func (o *Community) IsMemberInChatsBloomFilter(chatID string) bool {
+	if o.IsControlNode() || !o.channelEncrypted(chatID) {
+		return true
+	}
+
+	chat, ok := o.config.CommunityDescription.Chats[chatID]
+	if !ok || chat.MembersList == nil {
```

Comment:
If the memberlist is empty because of missing encryption keys, is it `nil` or just an empty map/splice?

Reply:
There are two cases when `MembersList` can be nil:
1) For non-encrypted channels
2) For communities controlled by clients that haven't updated to this version